modules/rostests/apitests/shell32/ShellExecuteEx.cpp

```diff
@@ -6,8 +6,82 @@
  */
 
 #include "shelltest.h"
+#include<stdio.h>
 
 #define ok_ShellExecuteEx (winetest_set_location(__FILE__, __LINE__), 0) ? (void)0 : TestShellExecuteEx
+#define ok_ShellExecuteEx1 (winetest_set_location(__FILE__, __LINE__), 0) ? (void)0 : TestShellExecuteEx1
+
+
+/* returns TRUE on success and FALSE on failure */
+/* Returns the OS Vendor String in VendorString if found */
+BOOL GetOSVendor(char* VendorString, int length)
+{
+    CHAR file_name[25] = {"OperatingSystem.txt"};
+    CHAR cmdline[100] = {"ver > "};
+    FILE *fp;
+    CHAR OSVendor[100] = {0};
+    CHAR *myreturn;
+    INT pos;
+    char *space_ptr;
+    INT iResult;
+
+    if(VendorString && (length >= 20))
```

Comment:
Missing space: `if ()`

Reply:
Maybe true, but is it worth the effort to fix for a regressions test?